clearcut_detection_backend/test/data/test_data/tiles_to_download.txt

```diff
@@ -0,0 +1,2 @@
+L1C_T36UYA_A010815_20190402T083714
```

Comment:
you need to create .json file with config for test, where you should put names of rasters, pathes from where download geojsons.